### ISSUE TITLE
Disk changing not working properly

### ISSUE THREAD

**Issue by n1ghttrap** [libretro/libretro-uae#237]
I haven't found a single game that will continue loading from a second disk, usually just stopping in its tracks right there.  Turrican 3 and Wolfchild are a couple examples.  I'm tried ejecting the 1st disk at the time of the swap request, ejecting, selecting second disk, and inserting, but it will not load as expected.  Can this be confirmed as working or broken by anyone else?  I've tried just loading ADF files as a start, having tried m3us.

**Comment by n1ghttrap** [comment]
Found out the problem; it doesn't like it when I index disks from inside archives.  This might be something to fix in the future, if you care to.

**Comment by sonninnos** [comment]
Don't have multiple disks inside a single ZIP. The frontend does the extracting, and the last time I tried it also resets the core when selecting another disk, but now with RA 1.8.3 it does not even allow browsing inside the archive and just inserts the ZIP as-is, which would not work with multiple disks in a single ZIP even if the core had direct ZIP support, since it would not know which disk to put in which drive.

BTW only the cracked Turrican III is on two disks, so the IPF version resolves that problem also. M3U works fine too when the disks are extracted.

There was a problem in the disk changing, but it has been fixed for a while now. There is no need to return to the emulation in between "tray cycles" anymore in some games, like Flashback, which won't recognize the change if it happens without an empty state in between. 

Of course if running Lakka that may still be the case, since the core versions lag behind.
